packages/requisitions/src/RequestRequisition/api/api.ts

```diff
@@ -11,26 +12,51 @@ import {
   RequestRequisitionRowFragment,
   getSdk,
   RequestRequisitionFragment,
-  RequestRequisitionsQuery,
+  Sdk,
 } from './operations.generated';
 
 export type RequestRequisitionApi = ReturnType<typeof getSdk>;
 
-const requisitionParser = {
-  toDeleteInput: (line: RequestRequisitionRowFragment) => {
+const requestParser = {
```

Comment:
Added a couple functions into `requestParser` and also changed the `RequestQuery` object to use a closure for storeid/sdk rather than passing into individual functions